<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="word">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>931</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>46</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>word</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="alt1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>391</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>alternativ1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alt2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>391</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>alternativ2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alt3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>391</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>alternativ3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alt4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>460</y>
     <width>391</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>alternativ4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextQuestionButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:redarrow4.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
